### PR TITLE
Add team invitation link

### PR DESCRIPTION
**Pull request by vsaarinen**
This adds a dropdown menu to the top right corner of the UI along with a modal dialog that gives the user their team's signup link.

screenshot 2017-04-28 10 58 44

TODO:
- [x] Make whole option clickable instead of only the text in the dropdown
- [x] Improve copy/content
- [x] Improve design

### CODE REVIEW

**Comment by pekkapulli** [comment]
Nice! A few comments:
* Use team name instead of email for the menu. There's nothing in the email that indicates the team link can be found under it.
* Would be even nice to open the same menu from both the team name and the profile pic. If you do this, center the menu to the combined item.
* I'd make the options higher, 40px instead of 32px per item.
* Use the color variables and font size mixins for everything.
* Use the same stroke for the top arrow as you use for the box.

File: src/js/components/common/toggle-menu.scss [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  user-select: none;
+}
+
+.options {
+  position: absolute;
+  right: 0;
+  top: 2rem;
+  width: 200px;
+
+  color: #333333;
```
Comment (pekkapulli) [inline src/js/components/common/toggle-menu.scss]: $minard-dark-blue is the dark body text color

File: src/js/components/common/toggle-menu.scss [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  font-size: 1rem;
```
Comment (pekkapulli) [inline src/js/components/common/toggle-menu.scss]: font-size-M or font-size-MS

File: src/js/components/common/toggle-menu.scss [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  text-align: right;
+
+  background: #ffffff;
+  border: 1px solid $minard-light-gray;
+  border-radius: 4px;
```
Comment (pekkapulli) [inline src/js/components/common/toggle-menu.scss]: I feel this would be better with no round corners.

Comment (pekkapulli) [inline src/js/components/common/toggle-menu.scss]: Could be ok with 2px, too :)

**Comment by pekkapulli** [comment]
Looks better, but I'd still add 4px to the menu option height.

**Comment by dennari** [comment]
Great! Some thoughts:

- Instead of the stroke, which to me doesn't look as refined, could the box use the same kind of techniques as the project card?
screen shot 2017-05-02 at 20 44 50
- When not hovering, the spacing looks a bit inconsistent. I see two options: either have some kind of a delimiter between the selections visible all the time or use e.g. bold as the hover effect.
- Could the top-arrow in the box be the same one as the triangle icon next to the team name?

**Comment by vsaarinen** [comment]
Good idea about the project card styling. It looks better.

Regarding the second point: I think the current design without delimiters already looks better with >2 options (e.g. in #240). Should we keep it as is for now and change it in another PR if it still looks bad?

For the third point, what do you mean by using the same icon? They're currently the same size, especially now that the border was removed.

**Comment by dennari** [comment]
It's ok for me to go with the current spacings and improve later if needed. The third point was mainly about the fact that the triangle icon is rounded a bit whereas the top-arrow is sharp. I was just thinking if would be possible to use the exact same icon somehow.

**Comment by vsaarinen** [comment]
It might be somehow possible, but the box arrow is now done with [CSS trickery](http://www.cssarrowplease.com/) while the link caret is a Font Awesome icon. Making them the same is not trivial.

**Comment by pekkapulli** [comment]
Ahh one more comment, the team name link isn't touch-friendly as it's really narrow. Add padding?

I don't mind the arrow being different from the icon.

**Comment by vsaarinen** [comment]
Good point. I'll add the padding.

**Pull request merged**